vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -232,6 +232,26 @@ function Get-PrivatePackagesToCacheOnVHD {
     }
 }
 
+function Get-ToolsToVHD {
+    $cacheDir = "c:\toolscache"
```

Comment:
Keep same pattern to use a temp folder in $env:TEMP instead of creating a temp folder in the root?

Reply:
Put it into the map of vhdbuilder/packer/generate-windows-vhd-configuration.ps1 so it will use the folder `c:\akse-cache`